dmci/distributors/solr_dist.py

```diff
@@ -26,6 +26,11 @@
 
 logger = logging.getLogger(__name__)
 
+"""Configure log level for solrindexer
+NOTE: Should the be configureable from dmci config?
```

Comment:
The solrindexer will be installed in the dmci container, right? Then I guess the answer is yes.. We only have one log for the container, right?